### PR TITLE
Message

### PR DESCRIPTION
**Pull request by hrikrishnankr**
Hi,
 Please find the pull request for the branch message which has the following changes 
      1.  Added pop up box for messages.
      2.  Connected with firebase.
      3.  Added spinner before loading.

@nishintn

### CODE REVIEW

**Comment by mad-nishin** [comment]
@hrikrishnankr ,
Pls sync your branch with master.

**Comment by hrikrishnankr** [comment]
Now sync with master

@nishintn

File: js/actions/AppActions.js [hunk 1]
```diff
@@ -13,6 +13,17 @@ var AppActions = {
 	      data: data
 	    });
 	},
+	hover:function(){
+		AppDispatcher.dispatch({
+			type: 'ON_HOVER'
```
Comment (mad-nishin) [inline js/actions/AppActions.js]: Be specific for the action names.

**Comment by hrikrishnankr** [comment]
Changed the action type in both store and actions.

**Pull request merged**